recipes/repoze.lru/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "repoze.lru" %}
+{% set version = "0.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0429a75e19380e4ed50c0694e26ac8819b4ea7851ee1fc7583c8572db80aff77
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
+
+test:
+  imports:
+    - repoze
+    - repoze.lru
+
+about:
+  home: http://www.repoze.org
+  license: BSD
```

Comment:
Looks like this is BSD-3-Clause "Modification_Variant"

https://fedoraproject.org/wiki/Licensing:BSD#Modification_Variant

Never saw that before :-/

Reply:
Heh me neither. What do you suggest I change this to?

Reply:
Unless I missed it I could not find it in https://spdx.org/licenses/

I would go with `BSD Modification Variant` for now.